codalab/apps/newsletter/views.py

```diff
@@ -25,6 +25,10 @@ def _send_mail(context_data, from_email=None, html_file=None, text_file=None, su
 
 
 def newsletter_signup(request):
+    if not settings.MAILCHIMP_API_KEY:
+        template = "newsletter/404.html"
```

Comment:
can do `raise Http404()` here and it'll automatically go to the existing 404 template -- did we need a new one?

Reply:
Raises Http404() in latest push